.github/workflows/merge.yml

```diff
@@ -0,0 +1,37 @@
+name: Merge the current PR
+on:
+  repository_dispatch:
+    types:
+      - merge-command
+
+jobs:
+  automerge:
+    name: Merge
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Add the merge-requested label
+        uses: actions-ecosystem/action-add-labels@v1
+        with:
+          github_token: "${{ secrets.CI_TOKEN }}"
+          number: ${{ github.event.client_payload.github.payload.issue.number }}
+          labels: merge-requested
+
+      - name: Automerge
+        uses: pascalgn/automerge-action@v0.12.0
+        env:
+          GITHUB_TOKEN: "${{ secrets.CI_TOKEN }}"
+          MERGE_LABELS: "merge-requested,!hold"
+          MERGE_METHOD: "merge"
```

Comment:
You probably have to update this since the merge strategy is different w.r.t CrownLabs